flake.nix

```diff
@@ -1,144 +1,427 @@
 {
   description = "plutarch";
 
-  inputs.haskell-nix.url = "github:input-output-hk/haskell.nix?rev=4aeeba8d713d0b98c92c8c717df24da17d463c1d";
+  inputs.haskell-nix.url = "github:L-as/haskell.nix?ref=master";
   inputs.nixpkgs.follows = "haskell-nix/nixpkgs-unstable";
-  inputs.haskell-nix.inputs.nixpkgs.follows = "haskell-nix/nixpkgs-2105";
   inputs.flake-compat-ci.url = "github:hercules-ci/flake-compat-ci";
 
-  inputs.plutus.url = "github:input-output-hk/plutus?rev=6d8d25d1e84b2a4278da1036aab23da4161b8df8";
-  inputs.cardano-prelude.url = "github:input-output-hk/cardano-prelude?rev=fd773f7a58412131512b9f694ab95653ac430852";
+  # https://github.com/input-output-hk/plutus/pull/4328
+  inputs.plutus.url = "github:L-as/plutus?ref=master";
+  # https://github.com/input-output-hk/cardano-prelude/pull/162
+  inputs.cardano-prelude.url = "github:locallycompact/cardano-prelude?rev=93f95047bb36a055bdd56fb0cafd887c072cdce2";
   inputs.cardano-prelude.flake = false;
-  inputs.cardano-base.url = "github:input-output-hk/cardano-base?rev=0b1b5b37e305c4bb10791f843bc8c81686a0cba4";
+  inputs.cardano-base.url = "github:input-output-hk/cardano-base";
   inputs.cardano-base.flake = false;
   inputs.cardano-crypto.url = "github:input-output-hk/cardano-crypto?rev=07397f0e50da97eaa0575d93bee7ac4b2b2576ec";
   inputs.cardano-crypto.flake = false;
-  inputs.flat.url = "github:Quid2/flat?rev=d32c2c0c0c3c38c41177684ade9febe92d279b06";
+  # https://github.com/Quid2/flat/pull/27
+  inputs.flat.url = "github:Quid2/flat?rev=41a040c413351e021982bb78bd00f750628f8060";
   inputs.flat.flake = false;
-  inputs.Win32-network.url = "github:input-output-hk/Win32-network?rev=3825d3abf75f83f406c1f7161883c438dac7277d";
+  # https://github.com/input-output-hk/Win32-network/pull/10
+  inputs.Win32-network.url = "github:input-output-hk/Win32-network?rev=2d1a01c7cbb9f68a1aefe2934aad6c70644ebfea";
   inputs.Win32-network.flake = false;
+  # https://github.com/haskell-foundation/foundation/pull/555
+  inputs.foundation.url = "github:haskell-foundation/foundation?rev=0bb195e1fea06d144dafc5af9a0ff79af0a5f4a0";
+  inputs.foundation.flake = false;
+  # https://github.com/locallycompact/protolude
```

Comment:
This comment seems outdated. We are using upstream repo in fact.

Reply:
Which repository are you talking about?